src/app/components/grid/index.html

```diff
@@ -321,6 +324,7 @@
 			<ul>
 				<li>The list is presented in a tile, page section, or other container that might flow off the screen.</li>
 			</ul>
+			<p>Include the multiselect toolbar to allow users to select or clear all items and optionally view only the selected items.</p>
```

Comment:
I'd delete "optionally" and replace it with "to" since that's implicit from the fact that toolbar allows users to do it that this is optional. ... Also, it might just be me, but wording this sentence as a command seems odd here. It seems like it should just be a sentence about what the toolbar does: "The multiselect toolbar allows users to..."